unittests/AST/ASTImporterTest.cpp

```diff
@@ -2644,20 +2654,25 @@ class ImportFunctionsVisibility
   std::string getCode0() const { return std::get<0>(std::get<1>(GetParam())); }
   std::string getCode1() const { return std::get<1>(std::get<1>(GetParam())); }
   bool shouldBeLinked() const { return std::get<2>(std::get<1>(GetParam())); }
+
+  static decltype(functionDecl()) PatternF;
```

Comment:
Can't you have a simple non-static data member? Than it wouldn't be necessary to initialize the variable out-of-class.
```
auto PatternF = functionDecl(hasName("f")); // C++14
decltype(functionDecl()) PatternF = functionDecl(hasName("f")); // C++11
```

Reply:
I got error "field initializer is not constant", so this may work only with constructor.